device-registry/src/main/resources/db/migration/V5__system_info.sql

```diff
@@ -0,0 +1,7 @@
+
+CREATE TABLE DeviceSystem (
+  uuid char(36) NOT NULL,
+  system_info LONGTEXT NOT NULL,
```

Comment:
Actually, just checked with alaa and we are using mariadb also in prod/staging. So I think this should be a dynamic col, but not sure how well this will play with slick?

https://mariadb.com/kb/en/mariadb/dynamic-columns/

Probably using the connect engine https://mariadb.com/kb/en/mariadb/connect-json-table-type/ would be even better...